src/features/bivariate_manager/components/BivariateMatrixControl/components/BivariateMatrixHeadingEntry/BivariateMatrixHeadingEntry.tsx

```diff
@@ -1,11 +1,13 @@
 import { forwardRef, useCallback, useImperativeHandle, useRef } from 'react';
 import cn from 'clsx';
+import { ChevronDown16 } from '@konturio/default-icons';
 import DenominatorIcon from '../DenominatorIcon/DenominatorIcon';
 import { QuotientSelector } from '../QuotientSelector/QuotientSelector';
 import styles from './BivariateMatrixHeadingEntry.module.css';
 import type { CSSProperties, MouseEvent } from 'react';
 
 const getHeadingPositionStyle = (isColum: boolean, index: number) => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const styles: any = {
```

Comment:
Why you use any here?

Reply:
it just was used before, but ts don't allow to pass check

Reply:
After I remove this any locally ts not show any new errors

Reply:
![image](https://user-images.githubusercontent.com/6587585/192491449-68a35a8d-bdd2-4bd7-9834-661f70b62cbe.png)

Reply:
oh, I'll get rid of it, thanks! 🙏